docs/book/v1/installation.md

```diff
@@ -1,5 +0,0 @@
-**This is only a placeholder.**
```

Comment:
The file is required by MkDocs, otherwise the build will fail.

Reply:
Its not present in other packages?

https://github.com/mezzio/mezzio-swoole/tree/3.4.x/docs/book/v1
https://github.com/mezzio/mezzio-authentication-basic/tree/1.2.x/docs/book/v1
https://github.com/mezzio/mezzio-session-ext/tree/1.11.x/docs/book

Reply:
Correct because these documentations does not use the prepared installation page of the MkDocs theme.

The page is generated by the theme based on the configuration:

https://github.com/mezzio/mezzio-cors/blob/dc29fe824d9ced293d9ab3f819b6ee8af4960e47/mkdocs.yml#L13-L14